smtp-transport/src/main/kotlin/no/nav/emottak/plugin/Routes.kt

```diff
@@ -1,23 +1,68 @@
 package no.nav.emottak.plugin
 
+import arrow.core.Either
+import arrow.core.NonEmptyList
+import arrow.core.raise.either
+import io.ktor.http.HttpStatusCode
 import io.ktor.server.application.Application
 import io.ktor.server.application.call
+import io.ktor.server.auth.authenticate
 import io.ktor.server.response.respond
 import io.ktor.server.response.respondText
+import io.ktor.server.routing.Route
 import io.ktor.server.routing.get
 import io.ktor.server.routing.routing
 import io.micrometer.prometheus.PrometheusMeterRegistry
+import no.nav.emottak.AZURE_AD_AUTH
+import no.nav.emottak.PayloadRequestValidationError
+import no.nav.emottak.log
+import no.nav.emottak.repository.PayloadRepository
+import no.nav.emottak.repository.PayloadRequest
 
-fun Application.configureRoutes(registry: PrometheusMeterRegistry) {
+private const val REFERENCE_ID = "referenceId"
+
+fun Application.configureRoutes(registry: PrometheusMeterRegistry, db: PayloadRepository) {
```

Comment:
Jeg forstår at det kan være _fristende_ å forkorte variabelnavn for å gjøre koden kortere, men jeg tror det kan være fordelaktig å unngå forkortelser som kan være misvisende. For eksempel, å skrive `db`  for repository klasser kan være forvirrende. Kanskje vi heller kunne bruke `repository` fullt ut for å gjøre koden mer leselig og tydelig for alle?